gemhardware/include/gem/hw/amc13/AMC13Manager.h

```diff
@@ -138,6 +141,7 @@ namespace gem {
           mutable gem::utils::Lock m_amc13Lock;
 
           ::amc13::AMC13 *p_amc13;
+	  ::amc13::AMC13Simple *p_amc13simple;
```

Comment:
Please remove this, ::amc13::AMC13 inherits from ::amc13::AMC13Simple, so the extra pointer is unnecessary